lib/KBreadcrumbs.vue

```diff
@@ -1,378 +1,110 @@
-              </template>
-            </KRouterLink>
-            <span v-else>{{ crumb.text }}</span>
-          </li>
-
-          <li
-            v-else
-            :ref="`crumb${index}`"
-            :key="index"
-            class="breadcrumb-visible-item-last breadcrumbs-visible-item"
-          >
-            <span
-              class="breadcrumbs-crumb-text"
-              :style="{ maxWidth: lastBreadcrumbMaxWidth }"  
-            >
-              {{ crumb.text }}
-            </span>
-          </li>
-        </template>
-      </ol>
-    </div>
-  </div>
-
+      </KRouterLink>
+      <span v-else>{{ item.text }}</span>
+    </li>
+  </template>
+  <template #more="{ overflowItems }">
+    <KIconButton
+      size="small"
+      icon="chevronUp"
+      appearance="raised-button"
+    >
+      <template #menu>
+        <KDropdownMenu
+          :options="overflowItems"
+        />
+      </template>
+    </KIconButton>
+    <span>
+      >
+    </span>
+  </template>
+</KListWithOverflow>
 </template>
 
-
 <script>
-
-  import filter from 'lodash/filter';
-  import startsWith from 'lodash/startsWith';
-  import throttle from 'lodash/throttle';
-  import every from 'lodash/every';
-  import keys from 'lodash/keys';
-  import KResponsiveElementMixin from './KResponsiveElementMixin';
-
-  const DROPDOWN_BTN_WIDTH = 55;
-  const DEFAULT_LAST_BREADCRUMB_MAX_WIDTH = 300;
-
-  function validateLinkObject(object) {
-    const validKeys = ['name', 'path', 'params', 'query'];
-    return every(keys(object), key => validKeys.includes(key));
-  }
-
-  /**
-   * Used to aid deeply nested navigation of content channels, topics, and resources
-   */
-  export default {
-    name: 'KBreadcrumbs',
-    mixins: [KResponsiveElementMixin],
-    inheritAttrs: false,
-    props: {
-      /**
-       * An array of objects, each with a `text` attribute (String) and a
-       * `link` attribute (vue router link object). The `link` attribute
-       * of the last item in the array is optional and ignored. The `link`
-       * attribute can be set to `null` which will render the breadcrumb only
-       * as text, regardless of its position in the breadcrumb.
-       */
-      items: {
-        type: Array,
-        required: true,
-        validator(crumbItems) {
-          // All must have text
-          if (!crumbItems.every(crumb => Boolean(crumb.text))) {
-            return false;
-          }
-          // If link is truthy make sure it is a valid router link
-          return crumbItems.every(crumb => !crumb.link || validateLinkObject(crumb.link));
-        },
-      },
-      /**
-       * By default, the breadcrums will be hidden when the length of items is 1.
-       * When set to `true`, a breadcrumb will be shown even when there is only one.
-       */
-      showSingleItem: {
-        type: Boolean,
-        default: false,
-      },
+import KListWithOverflow from './KListWithOverflow.vue';
```

Comment:
Here we dont need to import there components, as they are already registered in the Vue context, we can use it without it.